ddpui/ddpprefect/prefect_service.py

```
@@ -751,3 +759,187 @@
+        f"workers/filter/",
+        {"work_queue_names": [work_queue_name], "work_pool_names": [work_pool_name]},
+    )
+
+    return res["count"]
+
+
+############################## Related to estimation of flow run times ##############################
+
+
+def compute_dataflow_run_times_from_history(
+    dataflow: OrgDataFlowv1, limit=20, statuses_to_include=["COMPLETED"]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)